sys-apps/policycoreutils/policycoreutils-2.4.ebuild

```diff
@@ -100,6 +100,7 @@ src_compile() {
 			INOTIFYH="$(usex dbus)" \
 			SESANDBOX="n" \
 			CC="$(tc-getCC)" \
+			PREFIX="${ROOT}" \
```

Comment:
Yeah, it (and things derived from it) are only referenced in the makefile, not the code.